tests/filecheck/dialects/seq/seq_ops.mlir

```diff
@@ -16,8 +16,8 @@ builtin.module {
   // CHECK: %compreg_poweron = seq.compreg %data, %clk powerOn %on : i14
   %compreg_all = seq.compreg %data, %clk reset %bool, %data powerOn %on : i14
   // CHECK: %compreg_all = seq.compreg %data, %clk reset %bool, %data powerOn %on : i14
-  %compreg_sym = seq.compreg sym #hw<innerSym@foo> %data, %clk : i14
-  // CHECK: %compreg_sym = seq.compreg sym #hw<innerSym@foo> %data, %clk : i14
```

Comment:
I’m sure we’ve seen this syntax grepping through circt test outputs. Do you mind if we keep both variants in tests?

Reply:
Looking at CIRCT tests, I see this new syntax, i.e.:
https://github.com/llvm/circt/blob/main/test/Dialect/Seq/compreg.mlir

Now, the previous syntax is the *qualified* one, that is, the one you should see for example in generic syntax.

I checked on similar situations against mlir-opt - which is using the same declarative syntax code - and it's only accepting the unqualified one when expecting it, not both - that would be extra tricky to get right!

I don't want to go over you on your dialect though; can you tell me if it solves the question for you?

Reply:
You‘re right – thanks for adding both syntaxes in checks, helpful.